src/contexts/WorkDataContext.js

```diff
@@ -0,0 +1,60 @@
+import { createContext, useState } from 'react';
+import { DateTime } from 'luxon';
+
+export const WorkDataContext = createContext();
+
+const createFormDate = () => {
+  return new DateTime.utc();
+};
+
+const workDataArray = Array(1)
+  .fill(createFormDate())
+  .map((item) => ({ formDate: item.toISODate() }));
+
+const workDataMap = new Map();
+
+workDataArray.map(
```

Comment:
this is bad practice.  Map's purpose is for mapping.  You're creating an array, and assigning it to nothing.  This could be:

`const workDataMap = new Map(workDataArray.map((data) => [parseInt(formDate.replace(/-/g, ''), data]));`